kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -47,53 +47,79 @@
 <script>
 
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import useUser from 'kolibri.coreVue.composables.useUser';
   import { mapGetters } from 'vuex';
+  import useUser from 'kolibri.coreVue.composables.useUser';
+  import { useLocalstorage, ref } from '@vue/composition-api';
```

Comment:
I think the `useLocalStorage` referred to here is https://vueuse.org/core/useLocalStorage/#uselocalstorage

Reply:
Also, please note the camel case 🙂